### ISSUE TITLE
Azure App Service Deploy Task Question - Multiple Sites

### ISSUE THREAD

**Issue by russelldyas** [Microsoft/vsts-tasks#5245]
Hello

I am sure I am missing something obvious.

Is there an easy way to deploy to multiple resources groups or sites with out having to create 100 tasks?

Our current setup...

A number of resources groups each have a separate portal in them. Need to create release process that deploys a new build to all the resource groups. I could create a separate task for each site however that becomes unmanageable after a few sites :)

As said I am sure I am missing something obvious.

From

Russell

**Comment by RoopeshNair** [comment]
You can explore multiple execution deployments section: https://docs.microsoft.com/en-us/vsts/build-release/concepts/process/phases#agent-phase or use Deployment Groups to run task in parallel on multiple agents at the same time.

**Comment by russelldyas** [comment]
@RoopeshNair 

I don't think that solves the issue as that will run same RM job several times.  But the RM process would still need to have one task for each website.  We want to be able to give the task a list of sites or resource group and single task deploys to those sites.

**Comment by KrishnaAdityaB** [comment]
This blog covers an example of multiple agents https://blogs.msdn.microsoft.com/jikuma/2016/12/13/run-phase-on-multiple-agents-in-parallel/
